test/tools/yulrun.cpp

```diff
@@ -88,7 +88,8 @@ void interpret(string const& _source)
 	InterpreterState state;
 	state.maxTraceSize = 10000;
 	state.maxMemSize = 0x20000000;
-	Interpreter interpreter(state);
+	shared_ptr<Dialect> dialect(EVMDialect::strictAssemblyForEVMObjects(langutil::EVMVersion{}));
+	Interpreter interpreter(state, *dialect);
```

Comment:
`test/tools/ossfuzz/yulFuzzerCommon.cpp` needs an update to reflect this API change.

Reply:
Also, `test/tools/ossfuzz/yulFuzzerCommon.h` needs a `#include <libyul/backends/evm/EVMDialect.h>`